languages/go/tests/unit_test.go

```diff
@@ -300,6 +302,38 @@ func TestRuleTypes(t *testing.T) {
 	}
 }
 
+func TestZeroValueRepr(t *testing.T) {
+	ffiPolar := ffi.NewPolarFfi()
+	host := host.NewHost(ffiPolar)
+	polarValue, err := host.ToPolar(Foo{})
+	if err != nil {
+		t.Fatalf("host.ToPolar failed: %v", err)
+	}
+	switch variant := polarValue.ValueVariant.(type) {
+	case ValueExternalInstance:
+		expected := "oso_test.Foo{Name: Num:0}"
+		if *variant.Repr != expected {
+			t.Errorf("Zero-value Foo{} repr didn't match.\n\tExpected: %v\n\tReceived: %#v", expected, *variant.Repr)
+		}
+	default:
+		t.Fatalf("Expected ValueExternalInstance; received: %v", variant)
+	}
+
+	polarValue, err = host.ToPolar(Foo{Name: "Zooey", Num: 42})
+	if err != nil {
+		t.Fatalf("host.ToPolar failed: %v", err)
+	}
+	switch variant := polarValue.ValueVariant.(type) {
+	case ValueExternalInstance:
+		expected := "oso_test.Foo{Name:Zooey Num:42}"
+		if *variant.Repr != expected {
+			t.Errorf("Zero-value Foo{} repr didn't match.\n\tExpected: %v\n\tReceived: %#v", expected, *variant.Repr)
```

Comment:
Super nit: error message is wrong. This is not the zero value case

Reply:
A _super nit_??? This calls for _Super Nit Man_!!!

Reply:
772c072